exec/cnex/global.c

```diff
@@ -114,9 +123,40 @@ static FILE *check_file(TExecutor *exec, void *pf)
     return f;
 }
 
-void global_init()
+void global_init(int argc, char *argv[], int iArgStart)
 {
     random_initModule();
+    VAR_args.address = cell_createArrayCell(0);
+    sys_initModule(argc, argv, iArgStart, VAR_args.address);
+    BuiltinVariables[0].value = VAR_args.address;
```

Comment:
Isn't `BuiltinVariables[0].value` already initialized to point to `VAR_args`? This assignment seems to be redundant.

Reply:
I thought so to, but strangely enough, if I remove that assignment, the code fails, and `BuildinVariables[0].value` always points to an empty Cell, of type cNone.  I was hoping to get your input on what might be happening here that I'm missing.

Reply:
Oh, well I see. `&VAR_args` and `VAR_args.address` are not the same thing.